Framework/Algorithms/test/DetectorEfficiencyCorTest.h

```diff
@@ -123,8 +123,8 @@ class DetectorEfficiencyCorTest : public CxxTest::TestSuite {
     std::vector<double> e{sqrt(5.0), sqrt(5.0), sqrt(5.0), 0.0};
 
     space2D->setBinEdges(0, x);
-    space2D->mutableY(0) = y;
-    space2D->mutableE(0) = e;
+    space2D->mutableY(0).assign(y.begin(), y.end());
+    space2D->mutableE(0).assign(e.begin(), e.end());
```

Comment:
Why assign if it is the full range?

You could also just do everything directly, `space2D->setHistogram(0, BinEdges{0.0, ...}, Counts{10, ...}, CountVariances{5.0, ...})`.